pages/tasks/task.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a good way to refactor the logic. The only change I would make would be to put this file in a folder called `pages/tasks/regions/` as it's not really a page, it's a page region. If you can make that change, and then squash the commits again this should be ready to merge.